src/domains/wallet/pages/ImportWallet/ImportWallet.tsx

```diff
@@ -104,6 +104,12 @@ export const ImportWallet = () => {
 				wallet = await activeProfile.wallets().importByMnemonic(value, network.coin(), network.id());
 			} else if (type === "address") {
 				wallet = await activeProfile.wallets().importByAddress(value, network.coin(), network.id());
+			} else if (type === "privateKey") {
+				wallet = await activeProfile.wallets().importByPrivateKey(network.coin(), network.id(), value);
+			} else if (type === "wif") {
+				const coin = await env.coin(network.coin(), network.id());
+				const privateKey = await coin.identity().privateKey().fromWIF(value);
+				wallet = await activeProfile.wallets().importByPrivateKey(network.coin(), network.id(), privateKey);
```

Comment:
I'll add a method tomorrow to directly import by WIF. It already exists but isn't exported.